plugins/woocommerce/changelog/e2e-remove-color-palette-iterations

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+
```

Comment:
Is it intentional that this is empty?

Reply:
Yes, in case of path/dev it is optional, I only updated code that's not being shipped.